bchrpc/server.go

```diff
@@ -640,18 +654,23 @@ func (s *GrpcServer) GetHeaders(ctx context.Context, req *pb.GetHeadersRequest)
 // GetTransaction returns a transaction given its hash.
 //
 // **Requires TxIndex**
+// **Requires SlpIndex for slp token metadata, and token metadata
```

Comment:
NIT: Would reword as `**Requires SlpIndex for all token metadata`

Reply:
done.